### DISCUSSION TITLE
Radio Nav page - Enter ILS by ident?

### DISCUSSION THREAD

**Discussion by cadilhac** [flybywiresim/aircraft#3591] [Q&A] [answered]
Currently at LEMH. On the Radio Nav page, I can enter the ILS by frequency (111.5), but if I enter the ILS ident (MAO), NOT ALLOWED is displayed. Is it normal?
Using dev branch.

**Comment by tracernz** [comment, marked as answer]
Work in progress to improve this: https://github.com/flybywiresim/a32nx/pull/3573
